MobiusCore/Source/Mobius.swift

```diff
@@ -19,7 +19,17 @@
 
 import Foundation
 
-public typealias Update<Model, Event, Effect> = (Model, Event) -> Next<Model, Effect>
+public struct Update<Model, Event, Effect> {
+    private let update: (Model, Event) -> Next<Model, Effect>
+
+    public init(update: @escaping (Model, Event) -> Next<Model, Effect>) {
+        self.update = update
+    }
```

Comment:
Do we also want a wrapper `init(update: @escaping(inout Model, Event) -> [Effect])`, or is that coming later? :-)

Reply:
I'd be happy to do a followup PR since more internals will likely change :)